docs/content/python/getting-started/quickstart/data/data.md

```diff
@@ -19,4 +19,47 @@ endswithURL: >
 expensesPath1: examples/quickstart/polar/expenses-01-python.polar
 expensesPath2: examples/quickstart/polar/expenses-02-python.polar
 isAllowed: is_allowed
+installation_new: |
+    ```bash
+    pip install --upgrade oso
+    ```
+    Or find the Oso package at [http://pypi.python.org/pypi/oso/](http://pypi.python.org/pypi/oso/).
+import: import
+import_code: |
+    ```python
+    from oso import Oso
+    oso = Oso()
+    oso.enable_roles()
+    ```
+example_app: a Flask
+load_policy: |
+    ```python
+    oso.load_file("authorization.polar")
+    ```
+getroles: get_roles
+classes: Python classes
+objects: Python objects
+methods: Python methods
+register_classes: |
+    ```python
+    oso.register_class(Page)
+    oso.register_class(User)
+    ```
+app_code: |
+    ```python
+    from flask import Flask
+
+    app = Flask(__name__)
+    @app.route("/page/<pagenum>")
+    def page_show(pagenum):
+        page = Page.get_page(pagenum)
+        if oso.is_allowed(
+            User.get_current_user(),  # the user doing the request
+            "read",  # the action we want to do
+            page,  # the resource we want to do it to
+        ):
+            return f"<h1>A Page</h1><p>this is page {pagenum}</p>", 200
```

Comment:
It gets a page object, but the page object doesn't have content, just an id. We could add content, something like:

Page 0: Alice's user page
Page 1: Alice's admin page
Page 2: Bob's admin page

This could help the user understand a bit more what permissions each page has. I'm not sure it's too necessary